src/seedu/addressbook/commands/EditCommand.java

```diff
@@ -38,8 +38,8 @@ public Offense getWantedFor() {
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ":\n"
             + "Edits the person identified by the NRIC number.\n\t"
-            + "Contact details can be marked private by prepending 'p' to the prefix.\n\t"
             + "Parameters: n/NRIC p/POSTALCODE s/STATUS w/WANTEDFOR [o/PASTOFFENSES]...\n\t"
+            + "p/POSTALCODE s/STATUS w/WANTEDOR o/PASTOFFENSES tags are optional. \n\t"
```

Comment:
This line of code is redundant. Please use square brackets to indicate optional tags.